tests/DataValues/DecimalValueTest.php

```diff
@@ -42,6 +42,7 @@ public function validConstructorArgumentsProvider() {
 		$argLists[] = array( '+0.0' );
 		$argLists[] = array( '+0.000' );
 		$argLists[] = array( '+1.0' . str_repeat( ' ', 124 ) );
+		$argLists[] = array( '4.2' );
```

Comment:
I didn't have a goal of making ' 4.2' (with space) valid, since no math function (like bcmath) would return such number.